packages/inline-banner/addon/components/nucleus-inline-banner.js

```diff
@@ -79,7 +78,7 @@ class NucleusInlineBanner extends Component {
   * @private
   */
   @computed('title', function () {
-    return htmlSafe(this.get('title'));
+    return this.get('title');
```

Comment:
This can be just `this.title`?

Reply:
I have removed the computed property now